### PR TITLE
Travis config update, remove branch aliases

### PR DESCRIPTION
**Pull request by dnsl48**
https://github.com/silverstripe/cwp-recipe-kitchen-sink/issues/46

P.S. broken tests are legit. same sniffs in master now: https://travis-ci.com/silverstripe/silverstripe-mfa/jobs/249087746

### CODE REVIEW

**Comment by ScopeyNZ** [comment]
The discussion about these PRs has been a bit stretched. If you aren't deleting `master` branch then can you leave the branch alias please. Or could you replace the `4.x-dev` alias with `5.x-dev` if you are planning on making a `4` branch?

**Comment by dnsl48** [comment]
done

File: .travis.yml [hunk 1]
```diff
@@ -17,19 +25,21 @@ env:
...
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev BEHAT_TEST=1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev NPM_TEST=1
+      env: DB=MYSQL BEHAT_TEST=1
     - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
```
Comment (robbieaverill) [inline .travis.yml]: Why have you removed these versions? This removes our ability to test against multiple versions of SilverStripe, which is a very important part of these build matrixes

File: composer.json [hunk 2]
```diff
@@ -42,12 +42,12 @@
...
-        "branch-alias": {
-            "dev-master": "4.x-dev"
-        },
         "expose": [
             "client/dist"
-        ]
+        ],
+        "branch-alias": {
+            "dev-master": "5.x-dev"
+        }
```
Comment (robbieaverill) [inline composer.json]: Can you put this back where it was? The reason we have it there is that you can delete it from patch branches without touching the `expose` config lines (in terms of git history). I know it's minor, but the only change required here is `4` -> `5`

**Pull request merged**